src/libraries/System.Net.Primitives/src/System/Net/NetworkCredential.cs

```diff
@@ -15,9 +16,9 @@ namespace System.Net
     /// </devdoc>
     public class NetworkCredential : ICredentials, ICredentialsByHost
     {
-        private string _domain;
-        private string _userName;
-        private object _password;
+        private string _domain = string.Empty;
```

Comment:
IL change. Either declare nullable or force initialization to null.

Reply:
Set to null.